src/components/_basic.less

```diff
@@ -72,3 +72,12 @@ a.sbtn {
 .purple-btn {
   .colorRule(@purple);
 }
+
+.black-btn {
+  .colorRule(@black);
+
+  &:focus {
+    box-shadow: 0px 0px 0 0.2rem fade(@lightText, 30%);
```

Comment:
Please don't use light text, use a new variable. Assign the variable the color `#7d7d7d` and fade percentage `67%`

Reply:
suggested change is added and branch is updated! please review! thank you!